docs/docs/references/virtual-views.mdx

```diff
@@ -0,0 +1,46 @@
+import CreateVirtualView from './assets/create-virtual-view.jpg';
+import UsingVirtualViews from './assets/using-virtual-views.jpg';
+import EditDeleteVirtualView from './assets/edit-delete-virtual-view.jpg';
+
+# Virtual Views
```

Comment:
seems short for a standalone article - maybe it could just be a new section in the SQL runner docs? if you think there will be a lot more detail to add in the future we can lave it separate too